packages/common/src/assets/testScenePreset.ts

```diff
@@ -537,7 +539,91 @@ export const testScenePreset = {
                   "type": "hemisphere-light"
               }
           ]
-      }
+      },
+      "09cc03c4-24e9-sd1b-bc2e-e7e742fb069f": {
+        "id": "09cc03c4-24e9-sd1b-bc2e-e7e742fb069f",
+        "entityId": "09cc03c4-24e9-sd1b-bc2e-e7e742fb069f",
+        "name": "Hemisphere Light",
+        "parent": "2266BED7-6CC4-48A6-95DD-9BCD3CF9EAFC",
+        "collectionId": "d4457fc0-24e4-11eb-bc2e-e7e742fb069f",
+        "index": 8,
+        "createdAt": "2020-11-12T02:14:45.000Z",
+        "updatedAt": "2021-04-02T00:35:15.000Z",
+        "components": [
+            {
+                "name": "transform",
+                "props": {
+                    "position": {
+                        "x": 0,
+                        "y": 10,
+                        "z": 0
+                    },
+                    "rotation": {
+                        "x": 0,
+                        "y": 0,
+                        "z": 0
+                    },
+                    "scale": {
+                        "x": 1,
+                        "y": 1,
+                        "z": 1
+                    }
+                },
+                "data": {
+                    "position": {
+                        "x": 0,
+                        "y": 10,
+                        "z": 0
+                    },
+                    "rotation": {
+                        "x": 0,
+                        "y": 0,
+                        "z": 0
+                    },
+                    "scale": {
+                        "x": 1,
+                        "y": 1,
+                        "z": 1
+                    }
+                },
+                "id": "09xx93c3-24e9-11eb-bc2e-e1e740fb069f",
+                "entityId": "1B698484-C15A-4CEC-9247-03873520DF70",
+                "createdAt": "2020-11-12T02:14:45.000Z",
+                "updatedAt": "2021-04-02T00:35:15.000Z",
+                "type": "transform"
+            },
+            {
+                "name": "reflectionprobe",
+                "props": {
+                    probePosition:new Vector3(0),
```

Comment:
these can just be { x: 0, y: 0, z: 0 } to match the rest of the json schema

Reply:
Ok sure